inc/functions_post.php

```diff
@@ -401,64 +401,81 @@ function build_postbit($post, $post_type=0)
 		}
 
 		// Display profile fields on posts - only if field is filled in
-		if(is_array($profile_fields))
```

Comment:
Well, I can revert to the old filter in `showthread.php`, but it will be buggy/inconsistent then since posts can be shown outside of `showthread.php` too. Even in the core - if you add a post through quick reply in 1.8.4, there won't be any profile fields in it because `newreply.php` is used for AJAX.

Any other ideas?